ksql-engine/src/test/java/io/confluent/ksql/analyzer/AnalyzerTest.java

```diff
@@ -235,8 +237,9 @@ public void shouldCreateCorrectSinkKsqlTopic() {
     analyzer.analyze(query, Optional.of(createStreamAsSelect.getSink()));
 
     Assert.assertNotNull("INTO is null", analysis.getInto());
-    final StructuredDataSource structuredDataSource = analysis.getInto();
-    final KsqlTopic createdKsqlTopic = structuredDataSource.getKsqlTopic();
+    final Optional<Into> into = analysis.getInto();
+    assertThat(into, is(not((Optional.empty()))));
```

Comment:
Is don't follow....

You suggested change is ... well, it looks wrong to me: its comparing a string to a boolean.

What do you mean by the 'remove the suppression'?

If this a case of review fatigue...? :D

Reply:
`MatcherAssert#assertThat(String reason, boolean assertion)` is what I was referring to. This will allow us to remove the suppression of "optional without isPresent check"